dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityOptions.nlsprops

```diff
@@ -429,10 +427,9 @@ generateaeskey.option-key.key=\
 \ \ \ \ --key=[key]
 generateaeskey.option-desc.key=\
 \tSpecify a key for use during AES encryption. This string is\n\
-\thashed to produce an encryption key which will be used to encrypt\n\
-\tand decrypt passwords. The encryption key is saved in base64 format\n\
-\tin a variable named wlp.aes.encryption.key. If this option is\n\
-\tomitted, a random key is be generated.
+\thashed to produce an encryption key which is used to encrypt\n\
+\tand decrypt passwords. If this option is omitted, a random key is\n\
+\tgenerated.
```

Comment:
Specifies a key that is used during AES encryption. This string is hashed to produce an encryption key that is used to encrypt and decrypt passwords. If this option is omitted, a random key is generated.